src/app/applications/applications.directive.app.js

```diff
@@ -3,12 +3,23 @@
 
   angular
   .module('roboconf.applications')
-  .directive('rbcfSingleApp', rbcfSingleApp);
+  .directive('rbcfSingleApp', rbcfSingleApp)
+  .directive('rbcfIconManagement', rbcfIconManagement);
```

Comment:
This is still not good.
Every directive must be defined in its own file. So, the **rbcfIconManagement** directive should be defined in (let's say) a **applications.directive.app-icon.js** file. :wink: 

This way, if we search a directive, we know which files contain a directive.
And given the name, we will most likely find it quickly. If one day we decide to delete it, deleting the file will be enough. No need to dig the code.